.changeset/cuddly-melons-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/features": patch
```

Comment:
I think we only need to release the "admin.branding.v1"?

Reply:
done